qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -50,6 +50,11 @@
 from qiita_core.exceptions import IncompetentQiitaDeveloperError
 import qiita_db as qdb
 
+if PY3:
```

Comment:
We only support python 2.7 in qiita so `PY3` will never be true; however, in the future we will like to move to python3 ... anyway, IMOO this should be removed.

Reply:
Agreed, will do.